market_nextdoor_api/migrations/0004_preorder_test_preorder_testitem.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 5.0 on 2024-01-06 23:18
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('market_nextdoor_api', '0003_alter_item_image'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Preorder_test',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
This looks great! I have a question here about BigAutoField for the id, curious what this is or what it is doing.

Reply:
I think that's the syntax for the migration to automatically add an id to each entry?